cli/provision.go

```diff
@@ -230,10 +230,10 @@ var cmdProvision = []cobra.Command{
 				}
 			}
 
-			flush(user)
-			flush(ut)
-			flush(things)
-			flush(channels)
+			logJSON(user)
+			logJSON(ut)
+			logJSON(things)
+			logJSON(channels)
```

Comment:
Maybe you can update `logJSON` function to receive list of JSON objects (objs...), so you can pass `users`, `ut`, `things` and `channels` in one function call.